setup.py

```diff
@@ -23,8 +23,8 @@
         "microcosm-logging>=1.5.0",
         "psycopg2-binary>=2.7.5",
         "pytz>=2018.5",
-        "SQLAlchemy>=1.2.11",
-        "SQLAlchemy-Utils>=0.33.3",
+        "SQLAlchemy>=1.4.10",
```

Comment:
SQLALchemy 1.4.10 fixes a bug (that was reported by us) affecting queries used by `microcosm-eventsource`